src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -46,6 +46,22 @@ public void Dispose()
             var server = new TcpListener(endPoint);
             server.Start();
 
+            _clientCertificateRemovedByFilter = false;
+
+            if (PlatformDetection.IsWindows7 && 
+                !useClientSelectionCallback && 
+                !Capability.IsTrustedRootCertificateInstalled())
+            {
+                // https://technet.microsoft.com/en-us/library/hh831771.aspx#BKMK_Changes2012R2
+                // Starting with Windows 8, the "Management of trusted issuers for client authentication" has changed:
+                // The behavior to send the Trusted Issuer List by default is off.
+                //
+                // In Windows 7 the DN list is sent within the Server Hello. If the client selection callback isn't 
+                // used and the client certificate's issuer is not in the server's Trusted Root Authorities the client,
+                // not send the certificate.
```

Comment:
Thanks! Re-arranging words per line caused me missing that last part. Still, reading this today, I feel that it can be improved.
I'll also change from Distinguished Names list (RFC naming) to Trusted Issuer List (MSDN naming).

```
In Windows 7 the Trusted Issuer List is sent within the Server Hello TLS record. On the server side, this list is built using certificates from the Trusted Root Authorities certificate store.
In all OS versions the client side will pick the first certificate that has an issuer found within the Trusted Issuer List.

A client certificate will not be sent (filtered-out) if:
1. The client selection callback is not used.
2. The server did not send the client certificate issuer within the Trusted Issuer List. 
```